package.json

```diff
@@ -5,11 +5,11 @@
   "main": "index.js",
   "scripts": {
     "start": "node index",
-    "prod-start": "NODE_ENV=production PLATFORM_API_BASE_URL=https://platform.nypl.org/api/v0.1 node index",
-    "dev-api-start": "PLATFORM_API_BASE_URL=https://dev-platform.nypl.org/api/v0.1 KMS_ENV=unencrypted node index",
-    "prod-api-start": "PLATFORM_API_BASE_URL=https://platform.nypl.org/api/v0.1 KMS_ENV=encrypted node index",
+    "prod-start": "cross-env NODE_ENV=production PLATFORM_API_BASE_URL=https://platform.nypl.org/api/v0.1 node index",
```

Comment:
cross-env does not need to be installed globally. It's added to our dependency list and npm will execute the cross-env's bin which then executes the cross-env process. What cross-env allows for is a windows and nix platform to execute the same cli env inputs.

Reply:
I see. Unless I've always misunderstood this, it seems this is a change in `npm run` that I missed. It automatically resolves executables to `./node_modules/.bin` ( https://docs.npmjs.com/cli/v6/using-npm/scripts#path ). In the past we've had to specify `./node_modules/.bin/...` or else it would resolve to `$PATH` (i.e. global executables). This is nice. And I'm glad we're moving to be less macos biased